### PR TITLE
Upgrade to python3.10

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
CC @andreafioraldi Fixes https://github.com/google/fuzzbench/issues/1490

File: presubmit.py [hunk 1]
```diff
@@ -241,14 +241,12 @@ def pytype(paths: List[Path]) -> bool:
     """Run pytype on |path| if it is a python file. Return False if it fails
     type checking."""
     paths = [path for path in paths if is_python(path)]
-    if not paths:
```
Comment (oliverchang) [inline presubmit.py:244]: Is this (and the print below) an intentional change?

**Review by oliverchang — APPROVED** [review]
LGTM, but should we also updated https://google.github.io/fuzzbench/getting-started/prerequisites/#python-programming-language ?

File: presubmit.py (hunk 1 above)
Comment (jonathanmetzman) [reply presubmit.py:244]: Yeah. We don't need this early bail out. The print wasn't intended though.

**Pull request closed without merging**